hello.go

```diff
@@ -77,8 +140,15 @@ func main() {
 		Region:   region,
 	}
 
+	eventsHandler := initEvents()
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		tmpl.Execute(w, data)
+		switch r.Method {
+		case http.MethodPost:
```

Comment:
would you consider an endpoint like `/event` instead of `/`?
that could make the application simpler.

Reply:
This won't work with people trying triggers out, as the default endpoint is `/`.

Reply:
Ack